scripts/pr_preview.js

```diff
@@ -0,0 +1,50 @@
+// @ts-check
+
+/** @param {import('@actions/github-script').AsyncFunctionArguments} _ */
+export function afterDeploy({ core, github, context }) {
+    /**
+     * @param {string} pr
+     * @param {string} ref
+     */
+    return async (pr, ref) => {
+        core.info(`Creating deployment event in PR #${pr} for '${ref}'…`);
+        const deployment = await github.rest.repos.createDeployment({
+            auto_merge: false,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            ref: `pull/${pr}/head`,
+            environment: `pr-preview-${pr}`,
+            transient_environment: true,
+            description: `Preview deployment for Pull Request #${pr}`,
+            required_contexts: [],
+        });
+
+        core.info('Marking deployment as complete…');
+        await github.rest.repos.createDeploymentStatus({
+            state: 'success',
+            environment_url: `https://pr-${pr}--ideditor-preview.netlify.app`,
```

Comment:
```suggestion
            environment_url: `https://pr-${pr}--ideditor.netlify.app`,
```
this needs to be changed before merging. The current URL is just for the demo

Reply:
Is there anything else that needs to be configured for this to work on the `ideditor.netlify.app` project? IIRC not, but let me know if there's something to change.

Reply:
URL changed. 

I don't think there's any extra configuration required, because I just created a netlify account yesterday, and the demo worked immediately with no settings changed in the netlify portal.